src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Dsa.cs

```diff
@@ -65,7 +65,7 @@ internal static int DsaKeySize(SafeDsaHandle dsa)
             // there were some leading zero bytes truncated due to BIGNUM semantics.
             // This minimizes the change the key size is wrong due to leading zero bits; there is only
             // a 2^64 chance that the key has 64 leading zero bits and thus the keysize will be wrong.
```

Comment:
That makes sense as I said I was assuming there was a 50% which if true then would have no guarantees about the strength of the key based on the key size passed in. I did remove part of that comment with last sync, but will tweak it again to remove reference to leading zeros.